### DISCUSSION TITLE
[Weekly Meeting] [2024-09-06] Meeting Minutes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1469] [Weekly Meetings]
- Recording
  -  No need to keep forever
  - Alternatives:
     - SVN
     - Youtube (good idea, but hard logistics now)
     - Attach to the discussions
     - Attach to the discussions only the transcripts

- Apache KIE 10 release
  - Image finally being able to be uploaded (large size images were blocked)
  - KIE Tools only missing
     - Were blocked until today due some failures
     - Release pipeline being adjusted and will be soon triggered

- One year without community release!

----
Recording

https://us06web.zoom.us/rec/share/Y64_636XXvWghJYa7iukSrheHAdpDZhpe0UwWdYSiUbrQQ1F-lMdgprTL5rwRUm7.uSSLaAXdSqaI2m-R

Passcode: $%yD71Q3

**Comment by porcelli** [comment]
No luck in attaching the full recording here 

<img width="672" alt="Screenshot 2024-09-06 at 9 17 42 AM" src="https://github.com/user-attachments/assets/b40cfd38-3a0b-4782-a375-01732e31e2b6">

**Comment by vsevel** [comment]
> One year without community release!

and will be strongly appreciated ;-) 
good luck
